packages/uhk-common/src/models/udev-rules-info.ts

```diff
@@ -3,14 +3,14 @@
  * Only on Linux need extra udev rules.
  */
 export enum UdevRulesInfo {
-    Unkonwn,
-    Ok,
+    Unkonwn = 'Unkonwn',
```

Comment:
This is misspelled. It should be `Unknown`.

Reply:
fixed